src/client/graphics/layers/AlertFrame.ts

```
@@ -0,0 +1,147 @@
+    if (!this.game) {
+      return; // Game not initialized yet
+    }
+
+    if (this.userSettings.alertFrame() && this.startTimestamp > 0) {
+      this.animTime = this.animTime + 1;
+      this.updateAnimation();
+    }
+
+    // Check for BrokeAllianceUpdate events
+    const updates = this.game.updatesSinceLastTick();
+    if (updates && updates[GameUpdateType.BrokeAlliance]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)